src/bio/smd/SConscript

```diff
@@ -7,7 +7,7 @@ def scons():
 
     denv = env.Clone()
 
-    prereqs.require(denv, 'argobots')
+    prereqs.require(denv, 'argobots', 'protobufc')
```

Comment:
is this needed?

Reply:
correct, it includes include/daos_srv/daos_server.h.
BIO actually needs it as well, we might do some cleanup in the future.